sdk/search/azure-search-documents/azure/search/documents/models/__init__.py

```diff
@@ -25,28 +25,20 @@
 # --------------------------------------------------------------------------
 
 from .._generated.models import (
-    Answers,
-    AnswerResult,
     AutocompleteMode,
     IndexAction,
     IndexingResult,
-    QueryLanguage,
     QueryType,
     SearchMode,
-    Speller,
 )
 from .._search_client import odata
 
 
 __all__ = (
-    "Answers",
```

Comment:
Should these removals also be documented? Or were they not really exposed?

Reply:
This is not a breaking change against last GA.

If customers complaint about missing the feature, we will make another beta (if not, it will be in July) and add these features back.

This is to align with services schedule. (some features are GAed while others are still in preview).